### ISSUE TITLE
Supporting Evidence accessibility audit

### ISSUE THREAD

**Issue by pacerwow** [department-of-veterans-affairs/va.gov-team#94613]
## Issue Description
Conduct an accessibility audit of the following pages:

supporting-evidence/orientation
supporting-evidence/service-treatment-records
supporting-evidence/service-treatment-records-upload
supporting-evidence/evidence-types
supporting-evidence/evidence-types-bdd
supporting-evidence/va-medical-records
supporting-evidence/private-medical-records
supporting-evidence/private-medical-records-upload
supporting-evidence/private-medical-records-release
supporting-evidence/additional-evidence
supporting-evidence/summary
how-claim-exams-work

---
## Tasks
- [ ] Conduct the accessibility audit as outlined in the Epic for the pages listed above
- [ ] Record bugs & violations in Google Sheets

## Acceptance Criteria
- [ ] An accessibility audit is performed for the pages listed above
- [ ] Results are documented in Google Sheets
- [ ] A summary is shared with our stakeholders during Sprint Review
